src/golang/lib/workflow/operator/connector/load_params.go

```diff
@@ -39,6 +39,50 @@ type S3LoadParams struct {
 	Format   string `json:"format"`
 }
 
+func CastToRelationalDBLoadParams(params LoadParams) (*RelationalDBLoadParams, bool) {
```

Comment:
If all we care about is whether the load param is a relational db load param, can we just cast it to `RelationalDBLoadParams` instead of trying to cast to the specific ones?

Reply:
Is this comment out-dated? It seems that this is exactly what the func is doing

Reply:
I can't do that directly because there is no generic RelationalDBLoadParamsWrapper and there was some weirdness when I was trying to make it a LoadParam. For now, I've decided to do something similar to https://github.com/aqueducthq/aqueduct/blob/377cd4d603b8160d1c647bb1149b63f8b3c0039c/src/golang/lib/workflow/operator/connector/extract_params.go#L70